packages/lib/src/artifacts/ZWeb3.ts

```diff
@@ -61,6 +65,14 @@ export default class ZWeb3 {
     return (await ZWeb3.accounts())[0];
   }
 
+  public static toChecksumAddress(address: string): string {
```

Comment:
Add a throw if checksum is incorrect. Logic should then be:

- If address seems checksummed (ie has at least an uppercase letter), validate it
  - If it is incorrect, throw
  - If it is correct, go
- If address has no uppercase letters, show a warning, and checksum it ourselves